parser-combinators.cabal

```diff
@@ -1,5 +1,5 @@
 name:                 parser-combinators
-version:              0.1.0
+version:              0.1.1
```

Comment:
PVP is for four component version numbers, this package follows semantic versioning A.B.C, if we add functionality in backward-compatible way we bump B.